src/components/DropdownProblem.tsx

```diff
@@ -31,14 +31,16 @@ export function buildClassName(response: string, correct: boolean, formDisabled:
 
```

Comment:
Oh -- a general comment I almost forgot: Let's make sure to update docs for users in `schemas.md` in this PR too that describes how to use this optional override.

Reply:
Added via latest commit.